pychromecast/discovery.py

```diff
@@ -116,26 +133,32 @@ def remove_service(self, _zconf, typ, name):
                     break
 
         if not cast_info:
-            _LOGGER.debug("remove_service unknown %s, %s", typ, name)
+            _LOGGER.debug("remove_service unknown %s, %s", type_, name)
             return
 
         if device_removed:
             self._cast_listener.remove_cast(uuid, name, cast_info)
         else:
             self._cast_listener.update_cast(uuid, name)
 
-    def update_service(self, zconf, typ, name):
+    def update_service(self, zc: zeroconf.Zeroconf, type_: str, name: str) -> None:
         """Called by zeroconf when an mDNS service is updated."""
-        _LOGGER.debug("update_service %s, %s", typ, name)
-        self._add_update_service(zconf, typ, name, self._cast_listener.update_cast)
+        _LOGGER.debug("update_service %s, %s", type_, name)
+        self._add_update_service(zc, type_, name, self._cast_listener.update_cast)
 
-    def add_service(self, zconf, typ, name):
+    def add_service(self, zc: zeroconf.Zeroconf, type_: str, name: str) -> None:
```

Comment:
Below we call the parameters, `zconf` and `typ`. Why do we change here?

Reply:
`_add_update_service` is not part of the `zeroconf.ServiceListener` interface, we can use any parameter name there.